test/integration/puppetlabs/services/jruby/jruby_metrics_service_test.clj

```diff
@@ -40,14 +41,11 @@
 ;;; Config / constants
 
 (def default-test-config
-  (assoc (jruby-testutils/jruby-puppet-tk-config
-           (jruby-testutils/jruby-puppet-config {:max-active-instances 2}))
+  (bootstrap-testutils/load-dev-config-with-overrides
+   {:jruby-puppet {:max-active-instances 2}
     :webserver {:port 8140
                 :host "localhost"}
-    :web-router-service {:puppetlabs.trapperkeeper.services.status.status-service/status-service "/status"}
-    :metrics {:server-id "localhost"}
-    :puppetserver {:ssl-client-header "X_ssl-client-header-FOO"
-                   :ssl-client-verify-header "X_ssl-client-verify-header-FOO"}))
+    :metrics {:server-id "localhost"}}))
```

Comment:
Yeah, probably.  Since this test is explicitly about metrics, though, I thought it would be better to have it in there.  Picking up configuration implicitly from the dev config that a specific test needs - e.g., number of JRuby instances used - seems more fragile to me.